test/test_common_prep.py

```diff
@@ -12,15 +12,15 @@ def prep():
 class TestCommonPrep(object):
     def test_convert_cell_to_int(self, prep):
         headers = ['test1', 'test2', 'test3']
-        row = dict(zip(headers, ['0', '1', '']))
+        row = dict(list(zip(headers, ['0', '1', ''])))
```

Comment:
I really don't think this list() is necessary here and throughout this file where you have this pattern. I tested a bit on my own and got the same output with and without the list. The one thing that the linter I use suggests is setting `strict=True` inside the zip.